_posts/2020-06-07-Human-Curation-and-Digital-Datasets.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Human Curation and Digital Datasets: A Problem in Multiple Parts"
+author: cgy
+date: 2020-06-07
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+One of the difficulties when creating and maintaining a human curated
+database is that humans are by nature error prone. In this article, I
+will explore some of the challenges and difficulties that the curators
+of the IrishGen database have encountered when dealing with ambiguous
+statements in the genealogies, how human error creeps in over time,
+and, finally, how [IrishGen](https://github.com/cyocum/irish-gen)
+attempts to avoid introducing new errors.
+
+This is not a new phenomenon within the area of the early Irish
+genealogies.  Even in the pre-modern period when scribes and redactors
+within the tradition were working, errors crept in over time.
+[Donnchadh Ó Corráin](http://isni.org/isni/0000000117710023) has
+suggested that early Irish scribes attempted to avoid errors by using
+a highly repetitious structure ('The Book of Ballymote: a Genealogical
+Treasure' in [Book of Ballymote: Codices Hibernenses Eximii
+II](http://www.worldcat.org/oclc/1046463513), pg. 10).  Ó Corráin also
+discusses in that chapter a few hypothesises which IrishGen is well
+placed to answer. Some of these errors were not even errors but
```

Comment:
Does one answer a hypothesis? Perhaps one tests a hypothesis and answers a proposition?